lldb/source/Core/ValueObject.cpp

```diff
@@ -615,7 +615,16 @@ bool ValueObject::GetSummaryAsCString(TypeSummaryImpl *summary_ptr,
       m_synthetic_value->UpdateValueIfNeeded(); // the summary might depend on
                                                 // the synthetic children being
                                                 // up-to-date (e.g. ${svar%#})
-    summary_ptr->FormatObject(this, destination, actual_options);
+    SummaryStatistics &summary_stats = GetExecutionContextRef()
+                                        .GetProcessSP()
+                                        ->GetTarget()
```

Comment:
This should be:
```suggestion
    if (auto target_sp = GetTargetSP())
      SummaryStatistics &summary_stats = target_sp->GetSummaryStatisticsForProvider(GetTypeName())
```
etc.
Not sure if we're guaranteed a `ProcessSP` and `TargetSP` here (which might be the cause of the PR test failures)

Reply:
Yes, please grab the target from GetExecutionContextRef() directly.